### PR TITLE
Update Guide for Extension Wallet

### PR DESCRIPTION
**Pull request by h0ngcha0**

### CODE REVIEW

File: docs/wallet/extension-wallet/overview.md [hunk 1]
```diff
@@ -4,12 +4,17 @@ title: Overview
...
-It also enables dApps to securely interact with the wallets, allowing dApp users to sign transactions,
-execute smart contracts, etc right in the browser.
+Alephium extension wallet helps users to store and manage their assets
+such as the native `ALPH` token, other fungible tokens as well as NFTs
+on the Alephium blockchain.  It also enables dApps to securely
+interact with the wallets, allowing dApp users to sign transactions,
+interact with the smart contracts, etc right from the browser.
 
-<img src={require("./media/landing.png").default} alt="Execute Transction Script" width="250" />
+<img src={require("./media/landing.png").default} alt="Execute Transction Script" />
```
Comment (polarker) [inline docs/wallet/extension-wallet/overview.md:13]: There is a zoom-in reset button in the screenshot 😅

**Review by polarker — COMMENTED** [review]
Maybe we could add a remark about account somewhere since some users are confused about it.

**Review by polarker — APPROVED** [review]
Great work!

File: docs/wallet/extension-wallet/overview.md (hunk 1 above)
Comment (h0ngcha0) [reply docs/wallet/extension-wallet/overview.md:13]: Updated :)

**Pull request merged**